src/lucky_avram/ext/lucky/input_helpers.cr

```diff
@@ -65,12 +65,13 @@ module Lucky::InputHelpers
                unchecked_value : String,
                checked_value : String,
                **html_options) : Nil forall T
+    attrs = [] of Symbol
     if field.param == checked_value
-      html_options = merge_options(html_options, {"checked" => "true"})
+      attrs = attrs | [:checked]
```

Comment:
this one could technically call `<<`, but I just kept it consistent with the other forms in this file.